web/public/mtg/index.html

```diff
@@ -113,6 +113,16 @@
         list-style: none;
         text-align: right;
       }
+
+      .option-row {
+        display: flex;
+        align-items: flex-end;
+        padding-left: 65px;
+      }
+      .level-badge {
```

Comment:
I'd add a bit of padding to the left, and try to center the badge names verticaly.

5 second fix: 
<img width="172" alt="image" src="https://user-images.githubusercontent.com/856034/189275029-74e1ad24-52f7-452a-a3f5-3088081846dd.png">

If you're ambitious, you can figure out how to Actually Center the badge lol